### ISSUE TITLE
Bug in nlp_bert_mcdropout experiment

### ISSUE THREAD

**Issue by XiaohanYa** [ElementAI/baal#195]
Hi,

Thank you for developing this great package.

I tried to run the nlp_bert_mcdropout experiment on Google colab, but I got the following error.
```ValueError: num_samples should be a positive integer value, but got num_samples=0```

Things I changed from your original code are as follows. I did so because I could not import the functions directly in colab.

```
from baal.active import get_heuristic
from baal.active.dataset.nlp_datasets import active_huggingface_dataset, HuggingFaceDatasets
from baal.active.active_loop import ActiveLearningLoop
from baal.bayesian.dropout import patch_module
from baal.transformers_trainer_wrapper import BaalTransformersTrainer

args = {"epoch": 100,
        "batch_size":32,
        "initial_pool": 1000,
        "model": "bert-base-uncased",
        "query_size": 100,
        "heuristic": "bald",
        "iterations": 20,
        "shuffle_prop": 0.05,
        "learning_epoch":20}
```
Could you please help me to resolve this problem? Thank you.

**Comment by Dref360** [comment]
Oh sorry about that, I fixed the notebook in #187  but not the export python script.

You can modify the script to

```python
    active_set = active_huggingface_dataset(raw_train_set, tokenizer)
    active_set.can_label = False # Need to manually do this for research
    valid_set = HuggingFaceDatasets(raw_valid_set, tokenizer)

    # We start labeling randomly.
    active_set.label_randomly(initial_pool)
```

**Comment by XiaohanYa** [comment]
Thank you

> Oh sorry about that, I fixed the notebook in #187 but not the export python script.
> 
> You can modify the script to
> 
> ```python
>     active_set = active_huggingface_dataset(raw_train_set, tokenizer)
>     active_set.can_label = False # Need to manually do this for research
>     valid_set = HuggingFaceDatasets(raw_valid_set, tokenizer)
> 
>     # We start labeling randomly.
>     active_set.label_randomly(initial_pool)
> ```

Thank you. It works now.
